crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -199,6 +230,17 @@ pub struct QuerySetState<T>(T);
 
 impl_query_set!();
 
+pub struct ParamSet<'w, 's, T: SystemParam> {
+    param_states: &'s mut T::Fetch,
+    world: &'w World,
+    system_meta: SystemMeta,
+    change_tick: u32,
+}
+
+pub struct ParamSetState<T>(T);
```

Comment:
A simple doc string here will help, since this is public and its use is not immediately obvious from the signature.

Reply:
done